server/service/osquery_utils/queries.go

```diff
@@ -1351,6 +1352,71 @@ var SoftwareOverrideQueries = map[string]DetailQuery{
 			return mainSoftwareResults
 		},
 	},
+	// deb_last_opened_at collects last opened at information from DEB package files on Linux
+	// hosts. Joining this within the main software query is not performant enough to do on the
+	// device, so we do it on the server instead.
+	"deb_last_opened_at": {
+		Query: `
+		SELECT package, MAX(atime) AS last_opened_at
+		FROM deb_package_files 
+		CROSS JOIN file USING (path) 
+		WHERE type = 'regular' AND regex_match(file.mode, '[1357]', 0)
```

Comment:
What's the `regex_match` doing here?

Also, how does this perform on hosts with a solid number of packages? Any concerns there, or does splitting the queries up clear things up enough, even for debs where the RPM files mode filtering optimization isn't available?